e2e/specs/st_arrow_dataframe_column_types.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("st.dataframe and st.data_editor support a variety of column types", () => {
+  before(() => {
+    // Increasing timeout since we're waiting for
+    // dataframe tables to be rendered.
+    Cypress.config("defaultCommandTimeout", 15000);
```

Comment:
Should we create different scripts? I feel like having one script with this might make this test a bottleneck and make it slow.

Reply:
I think we might not even need this timeout. I have removed it for now and test if there are any issues without the timeout